administrator/components/com_templates/views/template/tmpl/default_description.php

```diff
@@ -15,6 +15,6 @@
 	<?php echo JHtml::_('templates.thumbModal', $this->template->element, $this->template->client_id); ?>
 </div>
 <h2><?php echo ucfirst($this->template->element); ?></h2>
-<?php $client = JApplicationHelper::getClientInfo($this->template->client_id); ?>
```

Comment:
Ya know, PhpStorm is kinda funny about using namespaced classes in non-namespaced classes, it defaults to putting the full reference inline and not adding a use statement like it would in other cases.